Core/Systems/AltUseUISystem.cs

```diff
@@ -66,6 +66,6 @@ private void DrawChargeBar()
 		Main.spriteBatch.Draw(AltBar.Value, center, new Rectangle(0, 0, 52, 14), Color.White * _fadeBar, 0f, new Vector2(26, 7), 1f, SpriteEffects.None, 0);
 
 		Rectangle barSrc = new Rectangle(4, 14, (int)(44 * factor), 14);
-		Main.spriteBatch.Draw(AltBar.Value, center + new Vector2(4, 0), barSrc, Color.White * _fadeBar, 0f, new Vector2(26, 7), 1f, SpriteEffects.None, 0);
+		//Main.spriteBatch.Draw(AltBar.Value, center + new Vector2(4, 0), barSrc, Color.White * _fadeBar, 0f, new Vector2(26, 7), 1f, SpriteEffects.None, 0);
```

Comment:
Was this intended to be kept commented out?

Reply:
Forgot about that, suddenly the program got mad while hot reloading, commenting it out made it stop being mad; it wasn't related to what I was doing so idk what's going on there, am in bed now so uncomment it at your own behest(?).

Reply:
At work so couldn't test, but pushed the update

Reply:
Need someone to review again, right?

Reply:
Nah looks like I can merge

Reply:
Well, the code i wrote has not been reviewed; so was more of a "should probably"; but oh well.

Reply:
I looked at it 👍